styles.css

```diff
@@ -0,0 +1,195 @@
+
+/* General CSS rules for every viewport, including large desktop screens */
+body {
+    background-color: #615D6C;
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  footer {
+      color: #CAE5FF;
+  }
+
+  #topnav {
+    background-color: #615D6C;
+    overflow: hidden;
+  }
+  #topnav a {
+    float: left;
+    color: #CAE5FF;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+  }
+  #topnav a:hover {
+    background-color:  #CAE5FF;
+    color: #615D6C;
+  }
+
+  #topnav a.active {
+    background-color: lightskyblue;
+    color: white;
+  }
+
+  .home {
+    height: 80vh;
+    background-color: #615D6C;
+  }
+
+  .home_img {
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+  }
+
+  .about {
+    height: 80vh;
+    background-color: #615D6C;
+    color: #ACEDFF;
+  }
+
+  .about_img {
+    height: 30%;
+    width: 30%;
+    object-fit: cover;
+  }
+
+  .SOC {
+    background-color: #615d6c;
+    height: 100vh;
+    color: #ACEDFF;
+  }
+
+  .contact {
+    background-color: #615D6C;
+    color: #ACEDFF;
+    height: 100vh;
+  }
+
+  a.contact:visited {
+    color: #ACEDFF;
+  }
+
+  .logo {
+    object-fit: fill;
+    width: 100px;
+  }
+
+/* removes the underline under a link */
+  a:link {
+    text-decoration: none;
+  }
+/* the flex container becomes flexible by setting the display property to flex */
+  .flex-container {
+    /* display: flex;
+    flex-flow: row wrap;
+    height: 100vh;
+    justify-content: center;
+    align-items: center;
+    align-content: space-between; */
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .flex-container > div {
+      background-color: #cae5ff;
+      /* width: 40vh;
+      height: 55vh;
+      margin: 10px;
+      padding: 20px;
+      font-size: 17px; */
+      width: 300px;
+      height: 400px;
+      margin: 30px;
+     box-shadow: 5px 5px 20px black ; 
+      padding: 10px;
+      color: #615d6c;
+  }
+
+  .flex-item > h3 {
+      text-align: center;
+  }
+
+  .github-octocat {
+      max-width: 100%;
+  }
+
+  .subtext {
+    text-align: center;
+    font-size: 15px;
+    font-weight: 300;
+  }
+
+  /* Contact Form Style */
+  * {
```

Comment:
The wildcard selector is usually entered at the top of the stylesheet. The wildcard selects everything in your html therefore its great to keep at the top. Developers tend to start their stylesheets by setting margins and padding to 0 as some elements have default margins/padding as well as setting box-sizing: border-box. The comment may have someone look at this line of code and think it refers only to form.